doc/setup.md

```diff
@@ -41,6 +41,30 @@ PORT=3000 ASSETS_DEBUG=true ./script/start
 open http://localhost:3000
 ```
 
+## Seeding your dashboard with data
```

Comment:
Minor: how about "Populating with data", so that there isn't confusion that this has anything to do with `db/seeds.rb`?